next/pages/typography.css

```diff
@@ -0,0 +1,57 @@
+@import 'tailwindcss/base';
+@import 'tailwindcss/components';
+@import 'tailwindcss/utilities';
+
+@layer components {
+  .text-h1-lg {
+    @apply text-h-2xl lg:text-h-xxxl font-semibold;
+  }
+
+  .text-h1 {
+    @apply text-h-xl lg:text-h-2xl font-semibold;
+  }
+
+  .text-h2 {
+    @apply text-h-lg lg:text-h-xl font-semibold;
+  }
+
+  .text-h3 {
+    @apply text-h-md lg:text-h-lg font-semibold;
+  }
+
+  .text-h4 {
+    @apply text-h-default lg:text-h-md font-semibold;
+  }
+
+  .text-h5 {
+    @apply text-sm lg:text-h-default font-semibold;
+  }
+
+  .text-h6 {
+    @apply text-h-sm font-semibold;
+  }
+
+  .text-p1 {
+    @apply text-p-default lg:text-p-md;
+  }
+
+  .text-p2 {
+    @apply text-p-default;
+  }
+
+  .text-p3 {
+    @apply text-p-sm;
+  }
+
+  .text-p4 {
+    @apply text-p-xs;
+  }
+
+  .text-lbl-1 {
+    @apply text-p-md;
+  }
+
+  .text-lbl2 {
+    @apply text-p-default;
+  }
```

Comment:
yes, let's rename these as it's not really clear

as they shouldn't be needed that often, what about something really descriptive (and following the tailwind sizes name) ? `text-button-l` and `text-button-base` ?